lib/rules/lint-link-rel-noopener.js

```diff
@@ -18,9 +18,45 @@ var buildPlugin = require('./base');
  <a href="/some/where" target="_blank"></a>
  ```
  */
+
+var DEFAULT_CONFIG = {
+  regexp: /no(opener|referrer)/,
+  message: 'links with target="_blank" must have rel="noopener"'
+};
+
+var STRICT_CONFIG = {
+  regexp: /no(opener.*referrer)/,
+  message: 'links with target="_blank" must have rel="noopener noreferrer"'
+};
+
 module.exports = function(addonContext) {
   var LinkRelNoopener = buildPlugin(addonContext, 'link-rel-noopener');
 
+  LinkRelNoopener.prototype.parseConfig = function(config) {
+    var configType = typeof config;
+
+    var errorMessage = 'The bare-strings rule accepts one of the following values.\n ' +
+      '  * boolean - `true` to enable / `false` to disable\n' +
+      '  * array -- [`true`, `strict`] to enable validation for both noopener AND noreferrer\n' +
```

Comment:
Can we swap to `true`, `false`, and `'strict'` (e.g. use a single string for `"strict"` mode).